ztp/gitops-subscriptions/acm/clusters-app.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: clusters-ran-sub
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: Channel
+metadata:
+  name: clusters-ran-subscription
+  namespace: clusters-ran-sub
+  annotations:
+    apps.open-cluster-management.io/reconcile-rate: medium
+spec:
+  type: Git
+  pathname: 'https://github.com/openshift-kni/cnf-features-deploy'
```

Comment:
Should this be normally be pinned to a specific branch?

Reply:
the branch defined in the subscription CR not in the channel.